doc/book/features/container/factories.md

```diff
@@ -37,29 +37,29 @@ order to seed the `Application` instance:
 
 - `middleware_pipeline` can be used to seed the middleware pipeline:
 
-  ```php
-  'middleware_pipeline' => [
-      // An array of middleware to register.
-      [ /* ... */ ],
-      Zend\Expressive\Container\ApplicationFactory::ROUTING_MIDDLEWARE,
-      Zend\Expressive\Container\ApplicationFactory::DISPATCH_MIDDLEWARE,
-      [ /* ... */ ],
-  ],
+```php
+'middleware_pipeline' => [
+    // An array of middleware to register.
+    [ /* ... */ ],
+    Zend\Expressive\Container\ApplicationFactory::ROUTING_MIDDLEWARE,
+    Zend\Expressive\Container\ApplicationFactory::DISPATCH_MIDDLEWARE,
+    [ /* ... */ ],
+],
```

Comment:
Indentation was correct in the original. The renderer we use, mkdocs, indents the code blocks so that they properly appear under the bullet point to which they reference.